### PR TITLE
Wrap location with block element <div> so they don't word-break awkwardly in co-m-grid-table

### PR DESCRIPTION
**Pull request by sg00dwin**
Fixes https://jira.coreos.com/browse/CONSOLE-487

screen shot 2018-05-22 at 12 39 29 pm

### CODE REVIEW

File: frontend/public/components/service.jsx [hunk 1]
```diff
@@ -22,7 +22,7 @@ const menuActions = [Cog.factory.ModifyPodSelector, ...Cog.factory.common];
 const ServiceIP = ({s}) => {
   const children = _.map(s.spec.ports, (portObj, i) => {
     const clusterIP = s.spec.clusterIP === 'None' ? 'None' : `${s.spec.clusterIP}:${portObj.port}`;
-    return <span key={i}>{clusterIP}&nbsp;&nbsp;</span>;
+    return <div key={i}>{clusterIP}&nbsp;&nbsp;</div>;
```
Comment (spadgett) [inline frontend/public/components/service.jsx]: I think we can remove the non breaking spaces now

Comment (sg00dwin) [reply frontend/public/components/service.jsx]: removed them

**Comment by sg00dwin** [comment]
@spadgett do I need to manually merge this or will openshift-ci-robot merge it eventually?

**Comment by spadgett** [comment]
The ALM tests are failing, but it's a known issue. We can manually merge.

File: frontend/public/components/service.jsx [hunk 2]
```diff
@@ -22,7 +22,7 @@ const menuActions = [Cog.factory.ModifyPodSelector, ...Cog.factory.common];
 const ServiceIP = ({s}) => {
   const children = _.map(s.spec.ports, (portObj, i) => {
     const clusterIP = s.spec.clusterIP === 'None' ? 'None' : `${s.spec.clusterIP}:${portObj.port}`;
-    return <span key={i}>{clusterIP}&nbsp;&nbsp;</span>;
+    return <div key={i}>{clusterIP}</div>;
```
Comment (ggreer) [inline frontend/public/components/service.jsx]: These get wrapped in a `<p>` tag on line 28, but divs aren't allowed in paragraphs. Either we should use `<span>` with `<br />`, or the p should be changed to a div. I'm not sure how either option would affect styling.

**Pull request merged**